src/PresentationalComponents/Template/TemplateCard.js

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 import propTypes from 'prop-types';
-
-import { Button, Card, CardBody, CardHeader, CardFooter, Level, LevelItem, Title } from '@patternfly/react-core';
+import { Button, Card, CardActions, CardBody, CardHead, CardHeader, CardFooter, Level, LevelItem, Title } from '@patternfly/react-core';
```

Comment:
yer gonna hate me, gotta make these explicit, the individual imports rather than lumping em all together 😭 S EVIL WORK I know.. but s all for the better

Reply:
hahaha no worries at all :D